WarmUp1/Front22/src/index.ts

```diff
@@ -0,0 +1,21 @@
+/* 
+
+
+Given a string, take the first 2 chars and return the string with the 2 chars added at both the front and back, so "kitten" yields"kikittenki". 
+If the string length is less than 2, use whatever chars are there.
+
+
+front22("kitten") → "kikittenki"
+front22("Ha") → "HaHaHa"
+front22("abc") → "ababcab"
+
+*/
+
+export function front22(strParameter: string): string {
+  let charArray: string[] = strParameter.split("");
+  let TempChar = strParameter.slice(0, 2);
```

Comment:
Name is incorrect, and please make sure you use `const` instead of `let` by default